scaling_performance/optimizing_compute_resources.adoc

```diff
@@ -0,0 +1,65 @@
+[[scaling-performance-compute-resources]]
+= Optimizing Compute Resources
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+
+[[scaling-performance-overcomitting]]
+== Overcommitting
+
+You can use overcommit procedures so that resources such as CPU and memory are
+more accessible to the parts of your cluster that need them.
+
+Note that when you overcommit, you take the risk that another application may
+not have access to the resources it requires when it needs them, which will
+result in reduced performance. However, this may be an acceptable tradeoff in
+favor of increased density and reduce costs. For example, development, QA or
+test environments may be overcommited, whereas production might not be.
+
+OpenShift implements resource management through the compute resource model and
+quota system. See the documentation for more information about the
+xref:../dev_guide/compute_resources.html[OpenShift resource model].
+
+For more information and strategies for overcommiting, see the
+xref:../admin_guide/overcommit.html[Overcommitting documentation in the Cluster
+Administration Guide].
+
+
+[[scaling-performance-gold-image]]
+== Using a Gold Image to Improve Efficiency
+
+You can create a base {product-title} image with a number of tasks built-in to improve efficiency and reduce repetitive tasks.
+
+For example, because every node requires the `ose-pod` image in order to run pods, each node has to periodically connect to the Docker registry in order to pull the latest image. This can become problematic when you have one hundred nodes attempting this at the same time, and can lead to resource contention on the image registry, waste of network bandwidth, and increased pod launch times.
+
+To build a gold image:
+
+* Create an instance of the type and size required.
+* Ensure a dedicated storage device is available for Docker local image/container storage, separate from any persistent volumes for containers.
+* Fully update the system, and ensure Docker is installed.
+* Ensure the host has access to all yum repos.
+* link:https://access.redhat.com/documentation/en/red-hat-enterprise-linux-atomic-host/7/getting-started-with-containers/chapter-8-managing-storage-with-docker-formatted-containers[Set up thin-provisioned LVM storage].
+* Pre-seed your commonly used images (such as the rhel7 base image), as well as
+ {product-title} infrastructure container images (ose-pod, ose-deployer, etc.)
+ into your gold image.
+
+[[scaling-performance-debugging]]
+== Debugging {product-title} Using the RHEL Tools Container
+
+Red Hat distributes a container called the `rhel-tools` container. The purpose of this container is to:
```

Comment:
@stevekuznetsov @brenton We're hoping to get this live for the 3.5 release. If there's additions you'd like to see, please let me know and I can put them in now-ish. Always keen for feedback and ideas!